build-system/tasks/e2e/index.js

```diff
@@ -24,7 +24,16 @@ function e2e() {
   return gulp.src(config.e2eTestPaths, {read: false})
       .pipe(mocha({
         require: ['@babel/register', '../../../testing/e2e/test-module'],
-      }));
+      })
+          // stop serving on localhost:8000
+          .once('end', () => {
+            process.exit();
```

Comment:
`process.exit()` can sometimes prevent console output from being printed properly, say if you were using a reporter or a plugin to print test output.

Two points:
- I don't see where the server is being started. Is it in another file?
- Would it work if you replaced this with a call to quit the server, thereby allowing the parent `gulp` process to quit of its own accord?

Reply:
I run `gulp serve` in the body of the task declaration, but I see why it'd be better to run it outside. Will look at how visual-diff handles the server process.